airbyte/sources/registry.py

```diff
@@ -19,6 +22,87 @@
 _REGISTRY_ENV_VAR = "AIRBYTE_LOCAL_REGISTRY"
 _REGISTRY_URL = "https://connectors.airbyte.com/files/registries/v0/oss_registry.json"
 
+_LOWCODE_CONNECTORS_NEEDING_PYTHON = [
+    "source-alpha-vantage",
+    "source-amplitude",
+    "source-apify-dataset",
+    "source-avni",
```

Comment:
We have integration tests to assert these hard coded exclusions lists are correct and up-to-date. If we find the validation is too flaky (because of new connectors being frequently added/updated), we may choose to disable/remove those tests - or move to only running those checks on-demand.